config/admins.example.txt

```diff
@@ -9,5 +9,3 @@
 # If SQL-based admin loading is enabled, admins listed here will always be loaded first		  #
 # and will override any duplicate entries in the database.									  #
 ###############################################################################################
-Gomble = Host
```

Comment:
As a config, this file is tracked separately on the server so editing it in the codebase won't have any effect.

Reply:
Better to have it empty.

Reply:
Spring cleaning, makes the code look more neat for anyone in need of anyone using our code.

If it has no effect you will have no issues with it being cleaned up.

Reply:
That seems fine to me, better for distribution.